package.json

```diff
@@ -1,17 +1,23 @@
 {
   "name": "eslint-config-mavenlint",
-  "version": "1.0.3",
+  "version": "1.1.0",
   "description": "Mavenlink ESLint config",
   "main": "index.js",
   "repository": "https://github.com/mavenlink/mavenlint.git",
   "license": "MIT",
-  "peerDependencies": {
-    "eslint": ">= 3.7.0"
+  "devDependencies": {
+    "eslint": "^3.7.0",
+    "eslint-plugin-import": "^1.13.0",
+    "eslint-plugin-jsx-a11y": "^2.1.0",
+    "eslint-plugin-react": "^6.1.2"
   },
-  "dependencies": {
-    "eslint-config-airbnb": "^10.0.1",
+  "peerDependencies": {
+    "eslint": "^3.7.0",
```

Comment:
Problem was that using `>= 3.7.0` caused issues in that it kept complaining there wasn't a proper version of `eslint` installed.